Pod/Classes/Optional Data Sources/VOKFetchedResultsDataSource.m

```diff
@@ -223,24 +223,24 @@ - (void)tableView:(UITableView *)tableView didDeselectRowAtIndexPath:(NSIndexPat
 
 - (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView
 {
-    NSInteger sectionCount = [[_fetchedResultsController sections] count];
+    NSInteger sectionCount = _fetchedResultsController.sections.count;
```

Comment:
Does this still need to access the `_fetchedResultsController` or can it grab `self.fetchedResultsController`?